user_guide_src/source/libraries/validation/006.php

```diff
@@ -4,3 +4,8 @@
     'username' => 'required',
     'password' => 'required|min_length[10]',
 ]);
+// or
+$validation->setRules([
+    'username' => 'required',
+    'password' => ['required', 'min_length[10]'],
```

Comment:
I didn't realize we supported this format 😂 Good addition.

Reply:
With this format, probably we can use `|` in regex.